app/uk/gov/hmrc/gform/formtemplate/Substituter.scala

```diff
@@ -419,6 +419,16 @@ object Substituter {
         k.copy(key = k.key(substitutions), value = k.value(substitutions))
     }
 
+  implicit def descriptionTotalSubstituter[A](implicit
+    ev: Substituter[A, Expr],
+    ev2: Substituter[A, BooleanExpr]
+  ): Substituter[A, Option[AtlDescription.KeyValueBased]] = (substitutions, t) =>
+    t match {
+      case Some(kvPair) =>
+        Option(kvPair.copy(key = kvPair.key(substitutions), value = kvPair.value(substitutions)))
+      case _ => None
+    }
+
```

Comment:
```suggestion
  implicit def descriptionTotalSubstituter[A](implicit
    ev: Substituter[A, Expr],
    ev2: Substituter[A, BooleanExpr]
  ): Substituter[A, AtlDescription.KeyValueBased] = (substitutions, t) =>
    t.copy(key = t.key(substitutions), value = t.value(substitutions))
```

There is dedicated implicit for handling `Option` so you can just write ☝️

Reply:
thanks @josef-vlach!  Updated